lib/__tests__/package-integrity/integrity.test.ts

```diff
@@ -0,0 +1,107 @@
+import '@monocdk-experiment/assert/jest';
+import {
+  App, Stack,
+  aws_codebuild as codebuild,
+  aws_secretsmanager as sm,
+} from 'monocdk';
+import { LinuxPlatform, PackageIntegrityValidation } from '../..';
```

Comment:
I actually think tests should always import from the top level directory because it ensures these classes are actually being exported, i.e we haven't forgotten to add the to `index.ts`.